ala-ws-security-plugin/src/main/recources/ehcache3.xml

```diff
@@ -0,0 +1,20 @@
+<eh:config
```

Comment:
We need to document this config in the README in case the consuming app is already using ehcache and would need to merge this config into the existing config.